test/functional/feature_llmq_rotation.py

```
@@ -244,6 +259,66 @@ def run_test(self):
         assert_equal(rpc_qr_info["mnListDiffAtHMinus2C"]["baseBlockHash"], rpc_qr_info["mnListDiffAtHMinus3C"]["blockHash"])
         assert_equal(rpc_qr_info["mnListDiffAtHMinus3C"]["baseBlockHash"], genesis_blockhash)
 
+        self.log.info("Test 'quorum dkginfo' RPC upcoming DKG participation fields")
+        active_mn = self.mninfo[0]
+
+        assert_raises_rpc_error(-8, "lookahead_blocks must not be negative",
+                                 active_mn.get_node(self).quorum, "dkginfo", "", -1)
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)